core/src/main/scala/kafka/log/LogManager.scala

```diff
@@ -354,7 +354,7 @@ class LogManager(logDirs: Seq[File],
       addStrayLog(topicPartition, log)
       warn(s"Loaded stray log: $logDir")
     } else if (isStray(log)) {
-      // Unlike Zookeeper mode, which tracks pending topic deletions under a ZNode, KRaft is unable to prevent a topic from being recreated before every replica has been deleted.
+      // KRaft is unable to prevent a topic from being recreated before every replica has been deleted.
```

Comment:
I think simply deleting the zk statement is not enough, we should do a little rephrase for it.
How about like:
```suggestion
      // In KRaft, we are unable to prevent a topic from being recreated before every replica has been deleted.
```
The rest of statement we could leave it as what it is.

Reply:
addressed it